src/esp/bindings_js/index.js

```diff
@@ -28,12 +29,7 @@ function preload(url) {
 Module.preRun.push(() => {
   let config = {};
   config.scene = defaultScene;
-  for (let arg of window.location.search.substr(1).split("&")) {
-    let [key, value] = arg.split("=");
-    if (key && value) {
-      config[key] = value;
-    }
-  }
+  buildConfigFromURLParameters(config);
```

Comment:
It doesn't propagate everything passed in URL parameters yet. I agree with you that it is an important feature. Let me add that change in this PR as well.

Reply:
I talked with Mandeep and since we are using WindowlessEmscriptenApplication.js, logs are going into pre element with id log. It should be the right time to go ahead and move to EmscriptenApplication.js. I will make that change in separate PR. (If you want to debug, logs should be there in the pre element which is hidden).

Reply:
Ah, that's why :) Thank you.